app/controllers/campaign_dashboards_controller.rb

```diff
@@ -1,3 +1,5 @@
+# TODO: not being used
+
```

Comment:
Should we update with a deprecation?

Reply:
Only reason I didn't is bc I wanted to talk to Eric about it